prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -668,12 +686,33 @@ def build_transmission(method="sub2line", **kwargs):
         lines, substations, **island_kwargs
     )
 
+    # Separate DC lines from AC line processing, add ratings
+    ac_lines, dc_lines = split_lines_to_ac_and_dc(lines, const.dc_line_ratings.keys())
+    dc_lines["Pmax"] = dc_lines.index.to_series().map(const.dc_line_ratings)
```

Comment:
So the total number of dc lines we have is `len(const.dc_line_ratings)`, and we don't have any line with `TYPE in dc_types` but index is not in `const.dc_line_ratings`?

Reply:
That is correct. The function could identify DC lines based on the type without us knowing their capacity, it's just that in this particular application we do know.